packages/components/src/internal/components/administration/actions.ts

```diff
@@ -115,9 +115,17 @@ export const getGroupMembership = (groups: FetchedGroup[], groupMemberships): Gr
         const userDisplayName = curr['UserId/DisplayName'];
         const userDisplayValue = `${curr['UserId/Email']} (${userDisplayName})`;
         const memberIsGroup = !userDisplayName;
+        const foundGroup = groups.find(group => group.id === curr.UserId);
 
+        // Issue47306: When a member is not resolvable, do not accumulate the member into any groups.
+        // For example, if you are a Project Admin, a groupMembership row associating a site group with a user possessing no
+        // permissions in the project will result in your inability to resolve data on the permission-less user.
+        // That user will not be visible to you, and so should be excluded from the GroupMembership return value.
+        if (memberIsGroup && !foundGroup) {
+            return prev;
+        }
```

Comment:
as discussed, this approach works well and correctly filters out the unknown members from the given group. One thought that I had when reviewing this PR, is it possible to still add the unknown member to the group and just not resolve their "group name". That is, we don't shortcut the reducer function here but instead using a default string for the name like "<Unknown Group>" or something in the member object below. Would that then allow for us to retain the member count display in the group management page?

Reply:
Unfortunately it is not possible to tell whether the member is a group or a user—we only have the userid, and we would have to pull in data from core.users in order to ensure that the id is a user or not. So if we did have a default string, the counts would be inaccurate for the types of members. It still seems like something that might be worthwhile to preserve the counts, though—maybe we should talk to someone from Product or Design?

Reply:
I see what you are saying. That makes sense that we can't distinquish between a group member or a user member at that point. In that case, let's stick with what you currently have. Thanks